fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -578,7 +578,7 @@ protected void addLinkAndOptionsHttpHeaders(final FedoraResource resource) {
         if (resource.isMemento()) {
             options = "GET,HEAD,OPTIONS,DELETE";
         } else if (resource instanceof FedoraTimeMap) {
-            options = "POST,HEAD,GET,OPTIONS,DELETE";
+            options = "POST,HEAD,GET,OPTIONS";
```

Comment:
This looks good.
Could you additionally improve the "[FedoraVersioningIT.testOptionsTimeMap](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java#L1471-L1479)" test to check for a specific list of OPTIONS values?